GVFS/GVFS.Platform.Windows/ProjFSFilter.cs

```diff
@@ -64,7 +64,10 @@ public static bool TryAttach(ITracer tracer, string enlistmentRoot, out string e
                 if (result != OkResult && result != NameCollisionErrorResult)
                 {
                     errorMessage = string.Format("Attaching the filter driver resulted in: {0}", result);
-                    tracer.RelatedError(errorMessage);
+                    // This code is shared between GVFS.exe and GVFS.Service.exe

```

Comment:
This comment is going to be stale very soon because the processes will be renamed. But I also don't think it adds any real value here.